contentcuration/contentcuration/frontend/channelEdit/vuex/clipboard/actions.js

```diff
@@ -218,26 +291,29 @@ export function loadChannelColors(context) {
   }, Promise.resolve(Object.values(context.state.channelColors)));
 }
 
-/**
- * @param context
- * @param {string} node_id
- * @param {string} channel_id
- * @param {string|null} [target]
- * @param {boolean} [deep]
- * @param children
- * @return {*}
- */
-export function copy(context, { node_id, channel_id, extra_fields = {} }) {
+export function doCopy(context, { node_id, channel_id, extra_fields = {} }) {
   const clipboardRootId = context.rootGetters['clipboardRootId'];
   extra_fields[ClipboardNodeFlag] = true;
-
   // This copies a "bare" copy, if you want a full content node copy,
   // go to the contentNode state actions
-  return Clipboard.copy(node_id, channel_id, clipboardRootId, extra_fields).then(node => {
-    context.commit('ADD_CLIPBOARD_NODES', [node]);
-    // Refresh our channel list following the copy
-    context.dispatch('loadChannels');
-    return [node];
+  return Clipboard.copy(node_id, channel_id, clipboardRootId, extra_fields);
+}
+
+/**
+ * @param context
+ * @param {String} node_id
+ * @param {String} channel_id
+ * @param {Object} extra_fields
+ * @return {Promise<void>}
+ */
+export function copy(context, { node_id, channel_id, extra_fields = {} }) {
+  return context.dispatch('doCopy', { node_id, channel_id, extra_fields }).then(node => {
+    return context.dispatch('loadChannels', { id__in: [channel_id] }).then(() =>
+      context.dispatch('addClipboardNodes', {
+        nodes: [node],
+        parent: context.rootGetters['clipboardRootId'],
+      })
```

Comment:
Trigger loading channels and adding clipboard nodes after copy-- same for `copyAll` but in batch.